articles/container-instances/container-instances-tutorial-deploy-app.md

```diff
@@ -118,7 +122,8 @@ In this tutorial, you completed the process of deploying your containers to Azur
 [docker-windows]: https://docs.docker.com/docker-for-windows/
 
 <!-- LINKS - internal -->
+[az-container-create]: /cli/azure/container#az_container_create
 [az-container-show]: /cli/azure/container#az_container_show
 [az-group-delete]: /cli/azure/group#az_group_delete
 [azure-cli-install]: /cli/azure/install-azure-cli
-[prepare-app]: ./container-instances-tutorial-prepare-app.md
\ No newline at end of file
+[prepare-app]: ./container-instances-tutorial-prepare-app.md
```

Comment:
I don't see the `[prepare-app]` ref link referenced anywhere, so you can remove this line.

Reply:
I'd rather find the place where I wanted to reference it, which I did... Will propose a new version too.